.evergreen/ocsp/certs.yml

```diff
@@ -87,7 +108,7 @@ certs:
     subjectAltName:
       DNS: localhost
       IP: 127.0.0.1
-    authorityInfoAccess: 'OCSP;URI:http://localhost:9001/power/level,OCSP;URI:http://localhost:8100/status'
+    authorityInfoAccess: 'OCSP;URI:http://localhost:8100/status'
```

Comment:
I missed this the first time I generated the singleEndpoint variants, and have regenerated the associated cert as well. I realize now that technically we could probably skip this cert since `mongods` handle multiple endpoints without any issues, but it does no harm to have this cert.  I suppose a driver that does not support stapling but only supports a single endpoint could also use this cert for testing